src/managers/GuildMemberManager.js

```diff
@@ -237,6 +237,17 @@ class GuildMemberManager extends CachedManager {
 
     // Clone the data object for immutability
     const _data = { ...data };
+    if (_data.communicationDisabledUntil) {
+      if (_data.communicationDisabledUntil instanceof Date) {
+        _data.communication_disabled_until = _data.communicationDisabledUntil;
+        if(_data.communication_disabled_until.getTime() < Date.now()) throw new Error('COMMUNICATION_DISABLED_PAST_DATE');
```

Comment:
Discord accepts dates in the past currently (and doesn't reset on sending null), so the lib shouldn't throw